WordPress/build.gradle

```diff
@@ -266,10 +262,9 @@ android {
             dimension "buildType"
         }
 
-        // Also dynamically add additional `buildConfigFields` to our app flavors from any `wp.`/`jp.`-prefixed property in `gradle.properties`
-        def properties = loadPropertiesFromFile(checkGradlePropertiesFile())
-        addBuildConfigFieldsFromPrefixedProperties(wordpress, properties, ['wp'])
-        addBuildConfigFieldsFromPrefixedProperties(jetpack, properties, ['wp', 'jp']) // Inherit same properties used from WP first then overwrite with JP-specific ones
+        // Also dynamically add additional `buildConfigFields` to our app flavors from any `wp.`/`jp.`-prefixed property in `secrets.properties`
+        addBuildConfigFieldsFromPrefixedProperties(wordpress, rootProject.secretProperties, ['wp'])
+        addBuildConfigFieldsFromPrefixedProperties(jetpack, rootProject.secretProperties, ['wp', 'jp']) // Inherit same properties used from WP first then overwrite with JP-specific ones
```

Comment:
Thanks for spotting this! Actually, I think I'd prefer plain `secretProperties` instead of `rootProject.secretProperties` - the code navigation doesn't work in either case, so anyone looking for source will have to search project either way. And without `rootProject` prefix, it's all shorter and more readable.

Reply:
This makes sense too, since both are working fine, as long as we don't have a mixture of `secretProperties` and `rootProject.secretProperties` it is good with me! 👍